security-ldap/src/main/java/io/micronaut/configuration/security/ldap/LdapAuthenticationProvider.java

```diff
@@ -175,8 +187,11 @@ public LdapAuthenticationProvider(LdapConfiguration configuration,
                 }
             } finally {
                 contextBuilder.close(managerContext);
+                emitter.onComplete();
             }
         }, BackpressureStrategy.ERROR);
+        responseFlowable = responseFlowable.subscribeOn(scheduler);
```

Comment:
This variable is unnecessary. You can just use subscribeOn directly after the creation

Reply:
I had to extract the variable because when I tried: 

```java
return Flowable.create(emitter -> {
....
        }, BackpressureStrategy.ERROR).subscribeOn(scheduler);
```

It complained about `Required type` being `Publisher<AuthenticationResponse>` but provided `Flowable<Object>`.